requirements/common.in

```diff
@@ -3,110 +3,51 @@ gunicorn==20.0.4
 whitenoise[brotli]==5.0.1
 
 # Used by Whitenoise to provide Brotli-compressed versions of static files.
-Brotli==1.0.7
-Django==3.0.3
+Django==3.0.5
 celery==4.3.0
-kombu==4.6.3
 simplejson==3.17.0
 newrelic==5.2.1.129
 
 # Required by Django
 mysqlclient==1.4.6
-asgiref==3.2.3
-
-# Required by celery
-billiard==3.6.2.0
-pytz==2019.3
-
-# Required by kombu
-amqp==2.5.2
-
-# Required by amqp
-vine==1.3.0
 
 # Required by mozlog
-mozterm==1.0.0
-blessings==1.7
 jsonschema==3.0.1
 djangorestframework==3.11.0
 django-cors-headers==3.2.1
 mozlog==5.0
 
 # Required by djangorestframework's API documentation
 coreapi==2.3.3
-coreschema==0.0.4
-Jinja2==2.11.1
-MarkupSafe==1.1.1
-
-# Required by coreapi
-itypes==1.1.0
-uritemplate==3.0.1
-requests==2.22.0
-
-# Required by django.contrib.migrations
-sqlparse==0.3.0
 
 # Used directly and also by Django's YAML serializer.
 PyYAML==5.3
 django-environ==0.4.5
 
 # Whilst we don't use six ourselves any more, it's still required by a number of dependencies.
-six==1.14.0
-python-dateutil==2.8.1
-django-filter==2.2.0
-django-redis==4.11.0
-
-# Required by django-redis
-redis==3.4.1
-elasticsearch==7.5.1
-
-# required by requests and elasticsearch
-urllib3==1.25.8
-certifi==2019.11.28
-
-# required by requests
-chardet==3.0.4
-idna==2.8
+python-dateutil
+django-filter
+django-redis
 
 # required for taskcluster
-slugid==2.0.0
 taskcluster==27.0.0
-aiohttp==3.6.2
-mohawk==1.1.0
-async-timeout==3.0.1
-multidict==4.7.4
-yarl==1.4.2
-taskcluster-urls==12.1.0
 graphene-django==2.8.0
 
 # Used by graphene-django
-graphene==2.1.8
 graphql-core==2.2.1
-graphql-relay==2.0.0
 typing==3.7.4.1
-promise==2.3
-rx==1.6.1
-singledispatch==3.4.0.3
 iso8601==0.1.12
```

Comment:
Were you able to find where this package is used? I wasn't.

Reply:
I believe this could be removed. If you can handle it great if not we can tackle it another day.